styleguide/derek/global-components/basement/basement.ejs

```diff
@@ -1,15 +1,28 @@
+<% if(!hide_currency_selector) { var hide_currency_selector = true; } %>
```

Comment:
We're still missing one part to make this work.  We need one of the files to reference this.  As it's kind of trivial which page has this, I'd recommend just changing the the `/derek/incubation/office365.ejs` file to:

```
<%- partial("../_partials/_basement", {hide_currency_selector: false}) %>
```
That said, if @amberRrucker would prefer us having it on another page instead, we should double check with her on that.